### PR TITLE
Fix: Wizard styling buttons

### PR DESCRIPTION
**Pull request by capetillo**
### QUICK FIX: Wizard buttons position

Set the wizard buttons to have a position of fixed. For the screen recording, I changed `imagesPerRow` to be able to show that the buttons have a fixed position

**UPDATED AFTER LLOYD'S FEEDBACK**

https://github.com/LCOGT/datalab-ui/assets/54489472/555488b5-f4f8-4c1b-8228-ab1b01875aa2

Tablet views

https://github.com/LCOGT/datalab-ui/assets/54489472/385218fb-f63a-40c1-b63e-1286e332c78a

https://github.com/LCOGT/datalab-ui/assets/54489472/61cb59ba-b9e0-443b-92ea-e69e04d4dcc7

### CODE REVIEW

File: src/components/DataSession/OperationWizard.vue [hunk 1]
```diff
@@ -215,6 +215,7 @@ const handleThumbnailClick = (item) => {
 <style scoped>
 .wizard-background{
   background-color: var(--dark-blue);
+  height: 100vh;
```
Comment (LTDakin) [inline src/components/DataSession/OperationWizard.vue:218]: The buttons are in the right place but from the video it still looks like the wizard component is overflowing and we have to scroll. For the tablet we don't want to have to scroll at all but want to see everything on one screen

**Review by LTDakin — CHANGES_REQUESTED** [review]

File: src/components/DataSession/OperationWizard.vue (hunk 1 above)
Comment (LTDakin) [reply src/components/DataSession/OperationWizard.vue:219]: wasn't what I had in mind so actually we can go back to how it was before

**Review by LTDakin — APPROVED** [review]
can revert my request and it can be done another time, looks good though!

**Review by jnation3406 — COMMENTED** [review]
I probably missed this in another PR, but the images there should have a header/label saying input_images or whatever the field key is in plain english, so they have an idea what they are selecting.
